### ISSUE TITLE
Key is broken in version 2.1.4

### ISSUE THREAD

**Issue by raciat** [google-map-react/google-map-react#947]
**Describe the bug 🐛**

After upgrading to `google-map-react@2.1.4` the key is not being used to load Google Maps JS API. The URL is: `https://maps.googleapis.com/maps/api/js?callback=__googleMapsCallback`. Downgrade to `google-map-react@2.1.3` fixes the issue. 

**To Reproduce 🔍**

Code to reproduce:

```
<GoogleMap id="map" defaultZoom={14}
    bootstrapURLKeys={{ key: GOOGLE_MAPS_KEY }}
    defaultCenter={{ lat: latitudeGeo, lng: longitudeGeo }}
>
    <Marker lat={latitudeGeo} lng={longitudeGeo} match={matchGeo} />
</GoogleMap>
```

**Expected behavior 💭**

The key should be used while loading Google Maps JS API. The URL used to be: `https://maps.googleapis.com/maps/api/js?callback=_$_google_map_initialize_$_&key=##key##`, where `##key##` is the value passed through `GOOGLE_MAPS_KEY` variable.

**Comment by itsmichaeldiego** [comment]
@jpoehnelt Could you take a look at it?

**Comment by itsmichaeldiego** [comment]
@aviyadeveloper If you have the chance too!

**Comment by itsmichaeldiego** [comment]
@raciat Fixed and deployed in 2.1.5, sorry for that
